src/magic_folder/test/test_upload.py

```diff
@@ -156,7 +153,7 @@ def test_commit_a_file(self, mangled_name, content, upload_dircap):
                 mangled_name: [
                     u"dirnode", {
                         u"ro_uri": remote_snapshot_cap.decode("utf-8"),
-                        u"verify_uri": uri_from_string(
+                        u"verify_uri": tahoe_uri_from_string(
```

Comment:
This kind of just "hides" that we're still using the Tahoe API here? (I mean: we're importing from our own namespace, sure, but it's still relying on the Tahoe APIs here). I guess we can still search for "tahoe_uri_from_string" to get rid of it later, so meh ... but the "real" solution is `to_verify_capability()` or similar.

Bigger picture here is that _ideally_ there'd be a library (provided by Tahoe, ideally) that _just_ does capability-string manipulation so we're kind-of prototyping the API we want in `magic_folder.util.capabilities` essentially...

Reply:
Yeah, if this wasn't in test code, I'd have definitely add `to_verify_capability`. It probably makes sense to do that anyway.